parser.py

```diff
@@ -0,0 +1,25 @@
+import csv
+
+valid_sort_flags = {'N', 'A', 'L'}
+valid_sort_order = {'A', 'D'}
```

Comment:
You could add a comment calling out these are global constants and what they stand for

Reply:
Will do!